### PR TITLE
Imagine upgrade to 0.6.x 

### PR DESCRIPTION
**Pull request by nadar**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  | #7

Maybe this can help to force upgrading to imagine 0.6.x version.

### CODE REVIEW

File: BaseImage.php [hunk 1]
```diff
@@ -151,7 +155,20 @@ public static function crop($filename, $width, $height, array $start = [0, 0])
...
             ->crop(new Point($start[0], $start[1]), new Box($width, $height));
     }
-
+    
+    /**
+     * Rotate the image based on EXIF informations.
+     * 
+     * @param ImageInterface $image
+     * @param string $color
+     * @return \Imagine\Image\ImageInterface
```
Comment (samdark) [inline BaseImage.php]: Need `@since 2.0.5`

Comment (nadar) [inline BaseImage.php]: So you won't make a minor version release and stick with 2.0.x instead of 2.1.x?

Comment (cebe) [inline BaseImage.php]: no, it should be 2.1.0, not 2.0.5

Comment (samdark) [inline BaseImage.php]: Yeah. 2.1.0. Sorry for confusion.

**Comment by nadar** [comment]
@samdark @cebe Maybe its better to use `autorotate` instead of `autoRotate` as its also `thumbnail` and `watermark` instead of `thumbNail` & `waterMark`.

**Comment by nadar** [comment]
stupid squash commit messed everything up, thank you GitHub for providing an inline squash merge ability ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 ================================================
 
-2.0.5 under development
+2.1.0 under development
 -----------------------
 
-- no changes in this release.
+- Enh #20: Upgrading the imagine library from 0.5.x to 0.6.x. In order to upgrade to 0.6.x the color behavior had to be
+           changed. In addition a new `autorotate` method has been implemeneted in order to rotate images based in the
+           exif informations provided inside the image.
```
Comment (samdark) [inline CHANGELOG.md]: Need your github username here as in other CHANGELOG lines.

**Comment by samdark** [comment]
As for autorotate I think it makes sense to do use it all lowercase.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request closed without merging**